admin/HowToBuildDocs.md

```diff
@@ -0,0 +1,26 @@
+# How to Build Shield Documentation
```

Comment:
We use Material for MkDocs for implementation. It is better to add the description in this section that includes the line as follows.
https://squidfunk.github.io/mkdocs-material/getting-started/

Reply:
Added.